library/templates/dob_now_permits.yml

```diff
@@ -0,0 +1,35 @@
+dataset:
+  name: &name dob_now_permits
+  version: "{{ version }}"
+  acl: public-read
+  source:
+    script: *name
+    options:
+      - "AUTODETECT_TYPE=NO"
+      - "EMPTY_STRING_AS_NULL=YES"
+    geometry:
+      SRS: null
+      type: NONE
+
+  destination:
+    name: *name
+    geometry:
+      SRS: null
+      type: NONE
+    options:
+      - "OVERWRITE=YES"
+      - "PRECISION=NO"
+    fields: []
+    sql: null
+
+  info:
+    description: |
+      DOB NOW job applications received from DOB via FTP
+      the python handles the encoding of the source file which is not standard utf-8 and also ingesting
+      from the edm-private s3 bucket.
+      INSTRUCTION FOR DATA UPDATE:
```

Comment:
At the moment, this datasource is a special use case for dev-db (which is different from the data available on opendata). I think the description should be updated to mention the additional columns that we get from DOB and why these are important. Also because this is different dataset than what is on open data, should we think a little harder about the naming convention of the yaml?

Reply:
Good point. But I actually not sure what the additional columns are. Let me take a look into Devdb wiki to see if it is there

Reply:
I still think the "DOB NOW Permits received from DOB via FTP" isn't complete enough, this should explicitly mention the repo that sends the data to `edm-private/dob_now/dob_now_permits/`